src/Controller/RegistrationController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\RegistrationForm;
+use App\Entity\User;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class RegistrationController extends AbstractController
+{
+    /**
+     * @Route("/register", name="user_registration")
+     * @param Request $request
+     * @param UserPasswordEncoderInterface $passwordEncoder
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
+     */
+    public function register(Request $request, UserPasswordEncoderInterface $passwordEncoder)
+    {
+        // 1) build the form
+        $user = new User();
+        $form = $this->createForm(RegistrationForm::class, $user);
```

Comment:
```suggestion
        $form = $this->createForm(RegistrationForm::class, $user = new User());
```

I would do this instead but that is my personal perference.